notes.go

```diff
@@ -0,0 +1,98 @@
+package chartmogul
+
+// Note is the customer note as represented in the API.
+type Note struct {
+	UUID string `json:"uuid,omitempty"`
+	// Basic info
+	CustomerUUID string `json:"customer_uuid,omitempty"`
+	Type         string `json:"type,omitempty"`
+	Text         string `json:"text,omitempty"`
+	Author       string `json:"author,omitempty"`
+	CallDuration uint32 `json:"call_duration,omitempty"`
+	CreatedAt    string `json:"created_at,omitempty"`
+	UpdatedAt    string `json:"updated_at,omitempty"`
+}
+
+// UpdateNote allows updating note on the update endpoint.
+type UpdateNote struct {
+	Text         string `json:"text,omitempty"`
+	AuthorEmail  string `json:"author_email,omitempty"`
+	CallDuration uint32 `json:"call_duration,omitempty"`
+	CreatedAt    string `json:"create_at,omitempty"`
+	UpdatedAt    string `json:"updated_at,omitempty"`
+}
+
+// NewNote allows creating note on a new endpoint.
+type NewNote struct {
+	// Obligatory
+	CustomerUUID string `json:"customer_uuid,omitempty"`
+	Type         string `json:"type,omitempty"`
```

Comment:
Not sure if we should omit empty on these because it should never be empty (obligatory)